portfolio.css

```diff
@@ -4,6 +4,133 @@
 
 body {
     font-size: 62.5%;
+    font-family: 'Dosis', sans-serif;
+
+}
+/**** Nav Bar ****/
+
+.video_container {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    width: 100%;
+    height: 100%;
+    overflow: hidden;
+    min-height: 500px;
+}
+.video_container video {
+    /* Make video to at least 100% wide and tall */
+    min-width: 100%;
+    min-height: 100%;
+
+    /* Setting width & height to auto prevents the browser from stretching or squishing the video */
+    width: auto;
+    height: auto;
+
+    /* Center the video */
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%,-50%);
+}
+
+
+.nav_container {
+    display: flex;
+    position: relative;
+    width: 100%;
+    align-items: center;
+    flex-direction: column;
+    margin-bottom: 50vh;
+
+}
+
+
+.personal_logo {
+    width: 300px;
+    height: 300px;
+}
+
+.nav_bar {
+    font-size: 1.25rem;
+}
+
+
+/**** About me ****/
+
+.about_me_container {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    padding-top: 50px;
+    position: relative;
+}
+
+
+.profile_img {
```

Comment:
you'll need to position this instead of using margins if you want it to always appear on top of the video container